MetaMask/metamask-extension
# Update calculation for the Smart Swaps' Max fee

## **Description**

Before we were using a multiplier, now we do it dynamically. With the new approach the max fee shouldn't be bigger than 2x estimated gas fee.

## **Manual testing steps**
1. Open Swaps
2. Make sure Smart Swaps are enabled
3. Request a quote
4. See that Max fee is little below 2x estimated gas fee

## **Screenshots/Recordings**
![image](https://github.com/MetaMask/metamask-extension/assets/80175477/176edb85-353a-42a5-b488-82dc4ff73018)
